neuroscout/requirements.txt

```diff
@@ -46,8 +46,8 @@ Pillow==4.1.1
 psycopg2==2.7.1
 progressbar2
 py==1.4.33
-grabbit==0.2.3
--e git+https://github.com/bids-standard/pybids.git#egg=pybids
+grabbit==0.2.6
+pybids=0.7.0
```

Comment:
That should be an ==

Reply:
Also, that still produces this warning:
`pybids 0.7.0 has requirement grabbit==0.2.5, but you'll have grabbit 0.2.6 which is incompatible`

Reply:
0.7.0 *doesn't* require 0.2.6, it requires 0.2.5. The 0.2.6 update has been merged into master, but will be in 0.7.1. So probably the solution is to remove the explicit grabbit dependency and let pybid do its thing.